### ISSUE TITLE
CT x must be a finite number NaN

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/number-line-distance#8]
```
number-line-distance : xss-fuzz
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1597199894745/number-line-distance/number-line-distance_en.html?continuousTest=%7B%22test%22%3A%5B%22number-line-distance%22%2C%22xss-fuzz%22%5D%2C%22snapshotName%22%3A%22snapshot-1597199894745%22%2C%22timestamp%22%3A1597228940991%7D&brand=phet&ea&fuzz&stringTest=xss&memoryLimit=1000
Query: brand=phet&ea&fuzz&stringTest=xss&memoryLimit=1000
Uncaught Error: Assertion failed: x must be a finite number: NaN
Error: Assertion failed: x must be a finite number: NaN
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1597199894745/assert/js/assert.js:22:13)
at ArrowShape.lineTo (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1597199894745/kite/js/Shape.js:228:15)
at https://bayes.colorado.edu/continuous-testing/ct-snapshots/1597199894745/scenery-phet/js/ArrowShape.js:51:12
at arrayEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1597199894745/sherpa/lib/lodash-4.17.4.js:537:11)
at Function.forEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1597199894745/sherpa/lib/lodash-4.17.4.js:9359:14)
at new ArrowShape (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1597199894745/scenery-phet/js/ArrowShape.js:50:7)
at https://bayes.colorado.edu/continuous-testing/ct-snapshots/1597199894745/number-line-distance/js/common/view/DistanceShadedNumberLineNode.js:126:21
at listener (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1597199894745/axon/js/Multilink.js:45:18)
at TinyEmitter.emit (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1597199894745/axon/js/TinyEmitter.js:69:53)
at Vector2Property._notifyListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1597199894745/axon/js/Property.js:273:25)
id: Bayes Chrome
Snapshot from 8/11/2020, 8:38:14 PM
```

**Comment by jbphet** [comment]
I just checked CT and it looks clear.  @SaurabhTotey - have you already addressed this?

**Comment by SaurabhTotey** [comment]
That's odd, I haven't made any changes, so it must be happening only sometimes by chance. I will investigate into the cause of this issue.

**Comment by KatieWoe** [comment]
I did notice that it seemed to come and go, which is why I made the issue even though a few columns were clear.

**Comment by SaurabhTotey** [comment]
Since I am unable to manually reproduce, I made 4cd6e99 to try and see if that would fix the issue. CT still is having bad columns, so I will revert that commit and continue looking into what may be causing this.

**Comment by SaurabhTotey** [comment]
I believe that I finally found the cause of the issue. I had an arrow scale factor that was based off of the number line's range, but I forgot to add corresponding scale factors for the new ranges because I added more numberlines for the explore scenes. After adding in scale factors so that all numberlines can show arrows, this issue should be fixed. I will wait for a few columns on CT to show up green before closing this.
